### PR TITLE
Handle KeyRemoved

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-sdk/lib/api/DeployedWallet.ts [hunk 1]
```diff
@@ -139,4 +139,13 @@ export class DeployedWallet implements ApplicationWallet {
...
   }
+
+  subscribeDisconnected(onDisconnected: Function) {
+    const subscription = this.sdk.subscribe(
+      'KeyRemoved',
+      {contractAddress: this.contractAddress, key: this.publicKey},
+      onDisconnected,
+    );
+    return () => subscription.remove();
+  }
```
Comment (marekkirejczyk) [inline universal-login-sdk/lib/api/DeployedWallet.ts]: 👍

File: universal-login-react/src/ui/UFlow/Dashboard.tsx [hunk 2]
```diff
@@ -23,7 +23,9 @@ export const Dashboard = (props: DashboardProps) => {
   const deployedWallet = state.kind === 'Deployed' ? state.wallet : undefined;
 
   const [newNotifications, setNewNotifications] = useState<Notification[]>([]);
-  useEffect(() => deployedWallet && deployedWallet.subscribeAuthorisations(setNewNotifications), [deployedWallet]);
+  useEffect(() => deployedWallet?.subscribeAuthorisations(setNewNotifications), [deployedWallet]);
+
+  useEffect(() => deployedWallet?.subscribeDisconnected(() => walletService.disconnect()), [deployedWallet]);
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/UFlow/Dashboard.tsx]: That is a bit tricky code, can we make it more explicit?

**Comment by marekkirejczyk** [comment]
Very nice slick implementation

**Comment by xitronix** [comment]
@marekkirejczyk , @JustynaBroniszewska helped with live review :D

**Pull request merged**